code-challenges.test.js

```diff
@@ -72,13 +165,61 @@ var mixedDataArray1 = [3, "yo!", 94, -9, false, 0, 18, "hola!"]
 
 // b) Create the function that makes the test pass.
 
+//create a function named OnlySortedNumbers
+//takes in an argument that is an array
+const onlySortedNumbers = (array) => {
+    //crates a variable that stores the array and filters through it
+    var sortedNumArray = array.filter(value => {
+        // the filter will create a new array with only numbers
+        return typeof value === "number"
+    })
+// return the mapped array then sort it by using sort. Sort alone typically sorts strings but adding the function a,b allows numbers to sort
+return sortedNumArray.sort(function(a, b) {
```

Comment:
make sure to mind the indentation because indentation can be suggestive of how logic is structured (even more so in python 😝 ) 
```suggestion
    return sortedNumArray.sort(function(a, b) {
```